args/core/math/glm/detail/type_vec3.hpp

```diff
@@ -22,6 +22,13 @@ namespace args::core::math
 		typedef vec<3, T, Q> type;
 		typedef vec<3, bool, Q> bool_type;
 
+        static const type up;
+        static const type down;
+        static const type right;
+        static const type left;
+        static const type forward;
+        static const type backward;
+
```

Comment:
what is the use of these ? as far as I can tell these should be initialized here!

Reply:
since they're static they can't be initiliased here

Reply:
they are also const,

Reply:
ah well they aren't pod so I guess they can't be static constexpr, which would probably be the ideal case